src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -0,0 +1,34 @@
+import type { Address } from 'viem'
+
+import { MEMO_PART_DELIMITER } from './constants'
+
+export const addAggregatorAndDestinationToMemo = ({
+  quotedMemo,
+  aggregator,
+  destinationToken,
+  minAmountOut,
```

Comment:
Since this is used purely for long-tails, would probably make sense to have a paranoia check and throw if we have no minimums - we already have checks for this in consumers but things could change

Reply:
Ok why not, it's safer 👍